Ponca/src/Fitting/linePrimitive.h

```diff
@@ -0,0 +1,139 @@
+/*
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+
+#pragma onces
+
+#include "./defines.h"
+#include "./primitive.h" // PrimitiveBase
+#include <Eigen/Geometry>
+#include <Eigen/Core>
+
+namespace Ponca
+{
+
+
+
+template < class DataPoint, class _WFunctor, typename T = void  >
+class Line : public PrimitiveBase<DataPoint, _WFunctor>
```

Comment:
In the same way that `Plane` delegates data storage to `Eigen::Hyperplane` by inheritance:
https://github.com/poncateam/ponca/blob/78bcb73123b743f89bc2b913f8805fdda5a8153a/Ponca/src/Fitting/plane.h#L39-L40
`Line` could inherit Eigen::ParametrizedLine` (https://eigen.tuxfamily.org/dox/classEigen_1_1ParametrizedLine.html)